### PR TITLE
[release] Update version to 3.2-SNAPSHOT

### PR DESCRIPTION
**Pull request by PatrickRen**
This pull request bumps the version to 3.2-SNAPSHOT for master branch.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -62,7 +62,7 @@ limitations under the License.
     </distributionManagement>
 
     <properties>
-        <revision>3.0-SNAPSHOT</revision>
+        <revision>3.2-SNAPSHOT</revision>
```
Comment (leonardBang) [inline pom.xml:65]: Uh? should we also fix this in release-3.1 branch?

Comment (leonardBang) [inline pom.xml:65]: Uh? should we also fix this in release-3.1 branch?

**Review by leonardBang — APPROVED** [review]
Thanks @PatrickRen for the work, LGTM

File: pom.xml (hunk 1 above)
Comment (PatrickRen) [reply pom.xml:65]: I did and pushed directly on release-3.1 just now. We never noticed this before...

**Pull request merged**